hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/type/BasicNestedFieldsTest.java

```diff
@@ -76,7 +76,7 @@ public static Object[] parameters() {
     @BeforeClass
     public static void beforeClass() {
         Config config = smallInstanceConfig()
-                .setProperty(SQL_CUSTOM_TYPES_ENABLED.getName(), "true");
+                .setProperty(ClusterProperty.SQL_CUSTOM_CYCLIC_TYPES_ENABLED.getName(), "true");
```

Comment:
we should test first and foremost behavior in default config. I do not want to end up with situation when normal UDT usage (tested in this class) requires the flag. The flag should be enabled only in tests that really need it.

Reply:
`test_circularlyRecurrentTypes` uses cyclic UDT. I'll move it.

Reply:
so you can remove the property